QueryPartitionedCollection.md

```diff
@@ -0,0 +1,42 @@
+### How to Query Partitioned CosmosDB Collection
+
+With CosmosDB, you can configure [partition key](https://docs.microsoft.com/en-us/azure/cosmos-db/partition-data) for your collection.
+
+Below is an example about how to query partitioned collection with this spring data module.
+
+#### Example 
+
+Given a document entity structure:
+```
+    @Document
+    @Data
+    @AllArgsConstructor
+    public class Address {
+        @Id
+        String postalCode;
+        String street;
+        @PartitionKey
+        String city;
+    }
+```
+
+How to write the repository interface:
```

Comment:
better to have some description:  Add custom method with `And` operator to query..